### ISSUE TITLE
Hello, world! Debian 12 (Bookworm) JRE 17, Zotero and Noobies

### ISSUE THREAD

**Issue by RENANZG** [retorquere/zotero-deb#89]
I installed Debian 12 (Bookworm) and I wanted to use Zotero on it. Zotero works well, but the integration with Libre Office Writer is hampered by Java 17. The Writer ".otx" extension is only available for Java 11. When I try to install the extension manually, I get this error:

I installed Debian 12 (Bookworm) and I wanted to use Zotero on it. Zotero works well, but the integration with Libre Office Writer is hampered by Java 17. The Writer ".otx" extension is only available for Java 11. When I try to install the extension manually, I get this error: 

![screen-2023-06-16-13-55-40](https://github.com/retorquere/zotero-deb/assets/53377291/759f77f6-f6a6-4384-9596-720ffda1d763)

I tried to install the previous version but the same error still occurs:
user@debian:~$ java -version
openjdk version "11.0.19" 2023-04-18 LTS
OpenJDK Runtime Environment Corretto-11.0.19.7.1 (build 11.0.19+7-LTS)
OpenJDK 64-Bit Server VM Corretto-11.0.19.7.1 (build 11.0.19+7-LTS, mixed mode)
user@debian:~$ sudo update-alternatives --config java
Existem 2 escolhas para a alternativa java (disponibiliza /usr/bin/java).

  Selecção   Caminho                                        Prioridade Estado
------------------------------------------------------------
* 0            /usr/lib/jvm/java-11-amazon-corretto/bin/java   11100019  modo automático
  1            /usr/lib/jvm/java-11-amazon-corretto/bin/java   11100019  modo manual
  2            /usr/lib/jvm/java-17-openjdk-amd64/bin/java     1711      modo manual

![screen-2023-06-16-13-58-18](https://github.com/retorquere/zotero-deb/assets/53377291/f15c2ad9-1fff-4a8d-b067-15371bea14a0)

THANKS. HELP-M2

**Comment by retorquere** [comment]
Have you tried https://www.zotero.org/support/word_processor_plugin_troubleshooting#installation_error

**Comment by RENANZG** [comment]
> Have you tried https://www.zotero.org/support/word_processor_plugin_troubleshooting#installation_error

A miracle has occurred! What I did: 
1) I deleted my Libre Office settings: /home/ /.config/libreoffice/4/user 
2) I deleted the Zotero settings: /home/ /. Zother 
3) I put Java 17 back as default sudo update-alternatives --config java 
4) I installed Java Common sudo apt install libreoffice-java-common 
5) I started Zotero to do the automatic installation of the extension, but an error occurred:
![screen-2023-06-16-13-55-40](https://github.com/retorquere/zotero-deb/assets/53377291/4302ca6b-f365-4e78-80a0-9cdbaf209bac)
6) I took a deep breath
7) I started to do the manual installation of the ".otx" that I downloaded for Libre Office. I made a couple of tries and Writer was weird.
8) Libre Office started doing the automatic step-by-step and I was able to do the integration.
![screen-2023-06-16-14-40-53](https://github.com/retorquere/zotero-deb/assets/53377291/460d58d5-9e32-4c8a-ae49-d89b4fdf98c9)

I don't feel confident doing all my research, with Zotero software, in Debian 12. I'm too afraid of losing my research and notes. 

Thanks  a lot @retorquere and @zotero-deb. Problem solved !
